opennms-webapp/src/main/webapp/js/onms-requisitions/scripts/services/Requisitions.js

```diff
@@ -1184,6 +1185,36 @@
     };
 
     /**
+    * @description Gets the available locations.
+    *
+    * The 'Default' location is not considered as a valid option.
+    * The location field should be either null or a valid location with Minions.
+    *
+    * @name RequisitionsService:getAvailableLocations
+    * @ngdoc method
+    * @methodOf RequisitionsService
+    * @returns {object} a promise. On success, it provides a list of available locations.
+    */
+    requisitionsService.getAvailableLocations = function() {
+      var deferred = $q.defer();
+      var url = requisitionsService.internal.monitoringLocationsUrl;
+      $log.debug('getAvailableLocations: getting available locations');
+      $http.get(url)
+      .success(function(data) {
+        $log.debug('getAvailableLocations: got available locations');
+        var locations = [];
+        angular.forEach(data.location, function(loc) { if (loc['location-name'] != 'Default') locations.push(loc['location-name']); });
```

Comment:
If we exclude "Default" then it is impossible to assign a node to the "Default" location isn't it? The value needs to either be "Default" or ```null```. So it seems like we need to keep "Default" here (and make sure that it is the first entry in the list, followed by the rest in alphabetical order).